File: spring-data-requery-test-java/src/main/java/org/springframework/data/requery/domain/sample/AbstractConcreteType1.java
package org.springframework.data.requery.domain.sample;

import io.requery.Entity;

/**
 * AbstractConcreteType1
 *
 * @author debop@coupang.com
 * @since 18. 6. 25
 */
@Entity
public abstract class AbstractConcreteType1 extends AbstractMappedType {

    public AbstractConcreteType1() {}

    public AbstractConcreteType1(String attribute1) {
        this.attribute1 = attribute1;
    }

    private static final long serialVersionUID = -6990849517756084276L;
}

File: src/test/kotlin/com/coupang/springframework/data/requery/domain/sample/AbstractConcreteType1.kt
package com.coupang.springframework.data.requery.domain.sample

import io.requery.Entity

/**
 * com.coupang.springframework.data.requery.domain.sample.AbstractConcreteType1
 * @author debop
 * @since 18. 5. 30
 */
@Entity
abstract class AbstractConcreteType1(): AbstractMappedType() {

    constructor(attribute1: String?): this() {
        this.attribute1 = attribute1
    }
}